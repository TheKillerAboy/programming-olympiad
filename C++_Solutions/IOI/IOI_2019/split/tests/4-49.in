7d6697fb-2a55-4d69-94ee-21b41f37da6a
11 10
4 4 3
9 3
9 8
9 4
7 4
5 3
0 1
9 6
9 0
8 10
2 6
