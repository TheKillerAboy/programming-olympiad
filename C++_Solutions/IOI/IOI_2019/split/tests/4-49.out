9e46656f-28f5-4a1b-acb6-f2309f627891
OK
0 0 0 0 0 0 0 0 0 0 0
